package.json

```diff
@@ -44,11 +44,11 @@
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "npm run compile:src",
     "compile:src": "lerna run compile",
-    "compile:prod": "NODE_ENV=production node --max_old_space_size=2500 node_modules/webpack/bin/webpack.js --config webpack.prod.config.js -p",
+    "compile:prod": "NODE_ENV=production node --max_old_space_size=3000 node_modules/webpack/bin/webpack.js --config node_modules/terra-dev-site/src/webpack/webpack.config.js -p",
```

Comment:
Anyone have a good idea for aliasing out this file to something more friendly without having to include a webpack config locally?
`node_modules/terra-dev-site/src/webpack/webpack.config.js`

Reply:
maybe provide a terra-dev-site script that will wrap webpack such that it takes args ?.
Thinking

```compile:prod: 'NODE_ENV=production node --max_old_space_size=3000 tds-prode'```

which maps to 

tds-prod.js which calls 

```shelljs(webpack.js --config ../../src/webpack/webpack.config.js -p)```

Reply:
Actually should we be using the servestatic script here by chance?

Reply:
I've thought about wrapping, but I want to avoid wrapping webpack because i don't want ot duplicate all the different webpack config options that exist (there are a ton).

Serve static could be used, but creating the artifacts is a by-product of running the server and there are cases where we just want the build output (deploying to gh-pages).